civitai-downloader-v2/tests/unit/test_search_strategy.py

```
@@ -0,0 +1,393 @@
+#!/usr/bin/env python3
+"""
+Search Strategy tests.
+Tests for advanced search capabilities with filtering, sorting, and pagination.
+"""
+
+import pytest
+import sys
+from pathlib import Path
+from unittest.mock import Mock, patch, MagicMock
```

Ruff (0.12.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)